src/phoenix/server/api/helpers/playground_spans.py

```diff
@@ -224,12 +224,49 @@ def _input_value_and_mime_type(input: Any) -> Iterator[tuple[str, Any]]:
     yield INPUT_VALUE, safe_json_dumps(input_data)
 
 
-def _output_value_and_mime_type(output: Any) -> Iterator[tuple[str, Any]]:
-    yield OUTPUT_MIME_TYPE, JSON
-    yield OUTPUT_VALUE, safe_json_dumps(jsonify(output))
+def _merge_tool_call_chunks(
+    chunks_by_id: defaultdict[str, list[ToolCallChunk]],
+) -> Union[list[dict[str, Any]], dict[str, Any]]:
```

Comment:
```suggestion
) -> list[dict[str, Any]]:
```

Can we simplify the return type?

Reply:
yeah i can do the pulling of just one tool call elsewhere or probably just fine to leave it alist even if it's only one, just felt a bit odd

Reply:
i think it's alright to leave as a list. non-blocking